apps/server/Dockerfile

```diff
@@ -15,8 +15,11 @@ RUN rm -rf /app/packages/integrations/vercel
 #  Temp solution for bun
 RUN bun install --ignore-scripts
 
+# Compile the TypeScript application
+RUN bun build --compile --sourcemap apps/server --outfile=app
```

Comment:
You need to give the actual path of the file 
you should do --sourcemap src/index.ts but in the directory app server 

you should move this line to line 24

Reply:
> check once if it works 
@thibaultleouay